app/src/main/res/values/strings.xml

```diff
@@ -1020,4 +1020,12 @@ Do you want to switch to this account?</string>
   <string name="send_sensitive_information_safely">Send sensitive information, safely</string>
   <string name="import_saved_logins">Import saved logins</string>
   <string name="use_a_computer_to_import_logins">Use a computer to import logins from an existing password manager</string>
+  <string name="you_can_return_to_complete_this_step_anytime_from_settings">You can return to complete this step anytime Vault in Settings.</string>
```

Comment:
Should this be, "..anytime _in_ Vault Settings."?

Reply:
right you are, change I must 
<img src="https://media4.giphy.com/media/hqoInD6ATfvW0/giphy.gif"/>